src/Sdk/DTPipelines/Pipelines/ObjectTemplating/PipelineTemplateConverter.cs

```diff
@@ -400,14 +435,12 @@ internal static class PipelineTemplateConverter
             }
 
             // Fixup the if-condition
-            var isDefaultScope = String.IsNullOrEmpty(scope?.Value);
-            ifCondition = ConvertToIfCondition(context, ifToken, false, isDefaultScope);
+            ifCondition = ConvertToIfCondition(context, ifToken, false);
```

Comment:
The rest of the changes in this file are because legacy template-prototype code was removed (PR already approved on server, just got copied here)